pep-0697.rst

```diff
@@ -0,0 +1,357 @@
+PEP: 697
+Title: C API for Extending Opaque Types
+Author: Petr Viktorin <encukou@gmail.com>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 23-Aug-2022
+Python-Version: 3.12
+
+
+Abstract
```

Comment:
If you prefer, instead of manually specifying the highlighting language of each of your code blocks, since this PEP is exclusively C-related, you could just do this:

```suggestion
Python-Version: 3.12


.. highlight:: c


Abstract
```

Reply:
I do as well, FWIW—just wanted to offer that as an option.